docs/common-envs.md

```diff
@@ -0,0 +1,22 @@
+# Common Environment Variables
```

Comment:
Shall we mention how to deduce env names from config structs?

Reply:
Yeah, that probably a good idea. Do you think `common-envs` is a good place for that?